app/Main.hs

```diff
@@ -70,3 +73,6 @@ config = Options
 
      <*> many (argument str (metavar "ARG"))
 
+     <*> switch ( long "safe"
```

Comment:
@CristhianMotoche I would like to suggest a few changes here:

- Turn safe mode on automatically when the schema file is present.
- Add an option `--schema` to override the default path of the schema file.
- Add a flag `--no-type-check` to turn safe mode off explicitly.

What do you think?

Reply:
@sestrella I agree. Perhaps, we can change `--no-type-check` with `--no-schema`.

Reply:
@CristhianMotoche sounds good :+1: